src/ToastrConfirm.js

```diff
@@ -19,14 +19,14 @@ class ToastrConfirm extends Component {
       cancelText,
       transitionIn,
       transitionOut,
-      hideCancel
+      disableCancel
     } = confirm.options;
 
     this.okText = okText || confirmOptions.okText;
     this.cancelText = cancelText || confirmOptions.cancelText;
     this.transitionIn = transitionIn || confirmOptions.transitionIn;
     this.transitionOut = transitionOut || confirmOptions.transitionOut;
-    this.hideCancel = hideCancel || confirmOptions.hideCancel;
+    this.disableCancel = disableCancel || confirmOptions.disableCancel;
```

Comment:
You need to check if the variable is not undefined else in case of false it will always assign to the `confirmOptions.disableCancel`